osu.Game/Screens/OnlinePlay/OnlinePlayComposite.cs

```diff
@@ -65,6 +66,9 @@ public class OnlinePlayComposite : CompositeDrawable
         [Resolved(typeof(Room))]
         protected Bindable<TimeSpan?> Duration { get; private set; }
 
+        [Resolved(typeof(Room), nameof(Room.QueueMode))]
```

Comment:
Not a big deal, but is the `nameof` portion here required for some reason? Doesn't seem like it should be.

Reply:
Not required, only required when the name of the property doesn't match the resolved name. It's a pretty hidden implementation detail...